readme.md

```diff
@@ -158,7 +158,7 @@ Once installation is working, try creating and requesting resources. Note that t
 
 * **Question:** At least one of these will throw an error. Which one? How can you get around this issue?
 
-* **Your Answer:** 
+* **Your Answer:** "season_count" didnt throw an error, but it didn't show when I specified to show "season_count" only. I could specify to exclude all other key pairs by using "-title" so only "season_count" would show up. Not sure this is the best workaround???
```

Comment:
Let me know if you continue getting this issue. You should encounter an error when you run `.create().select()`. You will have to re-request the information or use the `.save()` method to return the updated record.